lib/typhoeus/adapters/faraday.rb

```diff
@@ -25,7 +25,6 @@ class Typhoeus < Faraday::Adapter
         remove_method :setup_parallel_manager if method_defined? :setup_parallel_manager
       end
 
-      remove_method :call              if method_defined? :call
```

Comment:
With this line removed typhoeus seems to continue to work just fine according to https://github.com/lostisland/faraday/pull/745, and it seems to pass locally with v0.9 - v0.13. I cannot understand why it caused a problem, and I worry about removing it. Any ideas?